wsjcpp.yml

```diff
@@ -0,0 +1,32 @@
+wsjcpp_version: v0.0.1
+cmake_minimum_required: 3.0
+cmake_cxx_standard: 11
+name: nlohmann/json
+version: v3.7.3
+description: JSON for Modern C++
+issues: "https://github.com/nlohmann/json/issues"
+keywords:
+  - "c++"
+  - "json"
+
+required-libraries:
+  - pthread
```

Comment:
Why is pthread needed?

Reply:
Not needed for json - my fail. Later I will update PR